handel/src/cli/index.ts

```diff
@@ -57,6 +94,22 @@ function logFinalResult(lifecycleName: string, envResults: EnvironmentResult[]):
             }
             success = false;
         }
+
+        // insert log entry into dynamo log table
```

Comment:
As with the other function, let's move the putItem call into its own function in the dynamodb-calls module.